examples/advanced/friendship-bot.py

```diff
@@ -0,0 +1,87 @@
+"""
+Python Wechaty - https://github.com/wechaty/python-wechaty
+Authors:    Huan LI (李卓桓) <https://github.com/huan>
+            Jingjing WU (吴京京) <https://github.com/wj-Mcat>
+2020 @ Copyright Wechaty Contributors <https://github.com/wechaty>
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import os
+import asyncio
+from typing import Optional
+
+from wechaty import (
+    Wechaty, Contact, Friendship, FriendshipType
+)
+
+
+class MyBot(Wechaty):
+    """
+    listen wechaty event with inherited functions, which is more friendly for
+    oop developer
+    """
+    def __init__(self):
+        super().__init__()
+        self.busy = False
+        self.auto_reply_comment = "Automatic Reply: I cannot read your message because I'm busy now, will talk to you when I get back."
+
+    async def on_friendship(self, friendship: Friendship):
+        administrator = bot.Contact.load('admin-id')
```

Comment:
Can we define a admin-id as a global variable? Then it is easier to understand.

Reply:
`local variable` or `global variable` pattern depends on the developers. Even `admin-id` from `database` is all supported.  This is the only `contact-id` string here. I believe it's easier to understand.